UnityProject/Assets/Plugins/Zenject/Source/Install/CompositeScriptableObjectInstaller.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using UnityEngine;
+using Zenject;
+
+namespace Zenject
+{
+    [CreateAssetMenu(fileName = "CompositeScriptableObjectInstaller", menuName = "Installers/CompositeScriptableObjectInstaller")]

```

Comment:
This ``CreateAssetMenu`` line can be removed 
after a new ``[MenuItem("Assets/Create/Zenject/...]`` is created (in Editor/ZenMenuItems.cs)